### ISSUE TITLE
Add support to install, manage and set Hashlink VM versions.

### ISSUE THREAD

**Issue by TheHENOOB** [l0go/haxeget#15]
If there's a possibility, adding support for managing Hashlink installs is a cool addition (at least for me).

**Comment by l0go** [comment]
Yeah! I want this too. there is sadly no official hashlink binaries provided for macOS or Linux and I don't want to limit a feature just to Windows.
